wolfssh/test.h

```diff
@@ -116,14 +116,20 @@
     #define NUM_SOCKETS 5
 #else /* USE_WINDOWS_API */
     #include <unistd.h>
-    #include <netdb.h>
-    #include <netinet/in.h>
-    #include <netinet/tcp.h>
-    #include <arpa/inet.h>
-    #include <sys/ioctl.h>
     #include <sys/socket.h>
     #include <pthread.h>
-    #include <fcntl.h>
+    #ifdef  WOLFSSH_LWIP
```

Comment:
Perhaps this LWIP section should be in its own section above and not in the linux one?